megatron/fused_kernels/scaled_masked_softmax_cuda.cu

```diff
@@ -18,7 +18,9 @@
 #include <cuda.h>
 #include <cuda_runtime.h>
 #include <cuda_fp16.h>
+#ifndef __HIP_PLATFORM_HCC__
 #include <cuda_profiler_api.h>
```

Comment:
So we don't include this header file, but there's no code that depends on this header file that needs to be commented out?

Reply:
It was causing the below error:

/root/Megatron-DeepSpeed/megatron/fused_kernels/scaled_masked_softmax_hip.hip:22:10: fatal error: 'cuda_profiler_api.h' file not found
#include <cuda_profiler_api.h>
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated when compiling for gfx908.
ninja: build stopped: subcommand failed.
Traceback (most recent call last):